src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -2406,15 +2413,26 @@ def security_domain_download(cmd, client, hsm_name, sd_wrapping_keys, security_d
 
         certificates.append(sd_jwk)
 
+    # save security-domain backup value to local file
+    def _save_to_local_file(file_path, security_domain):
+        try:
+            with open(file_path, 'w') as f:
+                f.write(security_domain.value)
+        except:  # pylint: disable=bare-except
```

Comment:
any error to end user when save failed?

Reply:
These are the original implementations.

But you are right, we can show some error message here.